robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -42,6 +49,20 @@ def set_enable_course_connections(self):
                 return
         raise Exception("Clicking element 'Enable Course Connections' failed after multiple tries")
 
+    def select_backfill_checkbox(self):
```

Comment:
might be a good idea to check first the status of the checkbox. If it is already checked then u do not want to change the status. I am thinking of scenarios where the test case with this keyword gets executed multiple times in the same org.

Reply:
Yeah I thought about it. But this case is unique here. In this page, this checkbox enables the button to initiate a batch/queue kinda process. Once that is done and if the user moves out of the screen, the check box automatically flips to unchecked. So for every iteration this checkbox will load as unchecked avoiding the necessary to check for it